staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/time.go

```diff
@@ -180,7 +180,7 @@ func (t Time) MarshalQueryParameter() (string, error) {
 		return "", nil
 	}
 
-	return t.UTC().Format(time.RFC3339), nil
+	return t.UTC().Format(time.RFC3339Nano), nil
```

Comment:
Changing marshalling looks like a compatibility change that could mess with clients reading the values.

Reply:
see https://github.com/kubernetes/kubernetes/pull/92595#issuecomment-652160810